packages/blade/src/components/Icons/Volume1Icon/Volume1Icon.tsx

```diff
@@ -0,0 +1,26 @@
+import { Svg, Path } from '../_Svg';
+import type { IconComponent } from '..';
+import useIconProps from '../useIconProps';
+
+const Volume1Icon: IconComponent = ({ size, color }) => {
```

Comment:
What should we do about Volume1 & Volume2? 

IMO they can be renamed to VolumeLow & VolumeHigh 

<img width="223" alt="image" src="https://user-images.githubusercontent.com/35374649/210962631-33b61f7a-0672-4b26-8dfc-75281d0a2505.png">

Reply:
IMO VolumeIncrement and VolumeDecrement? that is usually i see on our macs, tv remote controllers, etc